src/inverseRobot.js

```diff
@@ -7,7 +7,21 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const oldRobot = Object.entries(robot);
+  const newRobot = {};
+
+  for (const part of oldRobot) {
```

Comment:
Why do we need to increase complexity (from O(n) to O(2n))? Object.entries + for of